django/utils/decorators.py

```diff
@@ -57,7 +67,7 @@ def bound_func(*args2, **kwargs2):
         # want to copy those. We don't have access to bound_func in this scope,
         # but we can cheat by using it on a dummy function.
 
-        @decorator
+        @decorate
         def dummy(*args, **kwargs):
             pass
         update_wrapper(_wrapper, dummy)
```

Comment:
There is a slight difference between the decoration using a iterable or repeating @decorate_method(..) multiple times, in the latter case the update wrapper is called for every decorating function, in the iterable case it is called only once. could this be an issue?